docs/en_US/Compressor/Benchmark.md

```diff
@@ -0,0 +1,88 @@
+To provide an initial insight into the performance of various channel pruning algorithms, 
```

Comment:
Suggest to use term `channel pruning` and `filter pruning` to represent different pruning methods:
channel pruning:  prune input channel
filter pruning: prune output channel
Our current pruner implemented as filter pruning,   we may support channel pruning in the future.

Reply:
> Suggest to use term `channel pruning` and `filter pruning` to represent different pruning methods:
> channel pruning: prune input channel
> filter pruning: prune output channel
> Our current pruner implemented as filter pruning, we may support channel pruning in the future.

Thx, fixed.